framework/src/interface/port/phy_port.rs

```diff
@@ -458,7 +458,7 @@ impl PmdPort {
 
     #[inline]
     pub fn mac_address(&self) -> MacAddr {
-        let mut address = MacAddr::new_from_slice(&[0; 8]);
+        let mut address = MacAddr::new(0, 0, 0, 0, 0, 0);
```

Comment:
any specific reason perf wise? I'm cool w/ it, but wondering.

Reply:
nope. I got rid of `new_from_slice` and replaced with `From<[u8; 6]>`. And this is just easier to write cuz you don't have to import the `From` trait.

also I realized that the old `new_from_slice` is in fact not safe. it would panic if you ever call it with a slice that's less than 6 bytes. so indirectly fixed a potential bug with this change.